web/admin/src/SessionBox.js

```diff
@@ -99,26 +100,31 @@ export class SessionBox extends Component {
 
     handleNewSession = () => {
       const current = this.state.showNewSession
-      this.setState({showNewSession: !current})
+      this.setState({showNewSession: !current, color: "#FAFAFA"})
```

Comment:
setting color as state like this doesn't separate concerns well.  Color is just a display concern.  If we change the color that displays in a certain situation (e.g whether some in put is valid, etc.), that situation is state, but the new color should just be a style change.